src/components/App/App.js

```diff
@@ -3,24 +3,36 @@ import NavBar from '../NavBar/NavBar';
 import { Route } from 'react-router-dom';
 import './App.css';
 import StatePicker from '../StatePicker/StatePicker';
-
+import { getBirds } from '../../apiCalls';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-
+      selectedState: '',
```

Comment:
Now that we are passing the state abbreviation in to the setBirds call in StatePicker, we might not need to save this? Would love to hear your thoughts!

Reply:
You're right, I think it's redundant